### ISSUE TITLE
损失函数

### ISSUE THREAD

**Issue by yjsdut** [YunYang1994/CodeFun#10]
想问作者关于损失函数，`prob_loss = tf.reduce_mean(tf.reduce_sum(prob_loss, axis=[1,2,3,4]))`代码中，这里不是很理解，想问下prob_loss是根据label得到的2维的tensor，那么后面的 axis=[1,2,3,4]是很么意思呢？prob_loss只有2维，为什么会是后面的1 2 3 4,实在不是很理解，希望楼主解答

**Comment by yjsdut** [comment]
啊，不好意思，我明白了，谢谢。但是博主，我实在不能理解你的置信度损失函数，首先respond_bbox和respond_bgd这两个参数，我就理解不了，这个您不是按照论文里的置信度损失函数写的吧，那您可以提供一下公式吗？？谢谢
